src/Avalonia.Controls/Border.cs

```diff
@@ -92,12 +95,63 @@ public override void Render(DrawingContext context)
 
             if (background != null)
             {
-                context.FillRectangle(background, rect, cornerRadius);
+                context.FillRectangle(Brushes.Transparent, rect);
             }
 
-            if (borderBrush != null && borderThickness > 0)
+            if (borderBrush != null)
             {
-                context.DrawRectangle(new Pen(borderBrush, borderThickness), rect, cornerRadius);
+                var pen = new Pen(borderBrush, 4);
+                var resolution = 5.0;
+                
+
+                Point pt(double x, double y)
+                {
+                    return new Point(x, y);
+                }
+
+                void DrawSector(Point pos, double radius, double degStart, double degEnd, double fromSize, double toSize)
+                {
+                    var points = new List<Point>();
+                    var degSteps = ((degEnd - degStart) / resolution);
+                    var sizeStep = (toSize - fromSize) / degSteps/2.0;
+                    var curStep = fromSize / 2.0;
+
+                    for (double i = degStart; i <= degEnd; i += resolution)//outer radius
+                    {
+                        points.Add(new Point(pos.X + Math.Cos(i / 180.0 * Math.PI) * (radius+curStep), pos.Y + (-Math.Sin(i / 180.0 * Math.PI)) * (radius+curStep)));
```

Comment:
We have bezier curves support, don't generate points manually

Reply:
Where is it support? RenderContext doesn't support the feature. But you can refactor the code manually

Reply:
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Visuals/Media/StreamGeometryContext.cs#L72

Reply:
Sorry, but it's very hard for me to understand how to StreamContext works. There is my test snippet that don't draw some Bezier.
```csharp
var stream = new StreamGeometry();
 var streamContext = stream.Open();
streamContext.BeginFigure(new Point(0, 0), false);
streamContext.CubicBezierTo(rect.Position, rect.Position + new Point(100, 20), rect.Position + new Point(40, 100));
streamContext.EndFigure(true);
context.DrawGeometry(Brushes.Transparent, new Pen(Brushes.Red, 2), stream);
```

Reply:
context is `IDisposable`.

Reply:
it doesn't change anything.

Reply:
Where are some examples of using StreamGeometry?

Reply:
Take a look at `RectangleGeometry.RectChanged()`.

https://github.com/AvaloniaUI/Avalonia/blob/dbd7ce7bad04faa2fa6262b8def146fd46f00eaa/src/Avalonia.Visuals/Media/RectangleGeometry.cs#L54-L65

Reply:
Thanks, I'll make changes as soon as.